README.md

```diff
@@ -1,4 +1,46 @@
-# scsi
+# bus-scsi
 
-TODO: Enter the cookbook description here.
+Expose SCSI disk information via automatic attributes under the `bus.scsi`
+namespace.
 
+## Requirements
+
+This cookbook requires Chef 12.7+ and libudev-183+.
```

Comment:
{{libudev-183+}} is only required on Linux, isn't it? Maybe it should be more explicit.